common/serial/arch/arch_linux.h

```diff
@@ -0,0 +1,22 @@
+#ifndef ARCH_LINUX__H
+#define ARCH_LINUX__H
+
+#include <arch.h>
+#include <stdio.h>
+
+#define TRACE() printf("%s:%d\n", __PRETTY_FUNCTION__, __LINE__)
```

Comment:
This should either be in the .cpp, or temporary left here until we can move it somewhere else.